core/controllers/reader.py

```diff
@@ -300,7 +300,127 @@ def get(self, exploration_id):
 
 
 class StorePlaythroughHandler(base.BaseHandler):
-    """Handles a useful playthrough coming in from the frontend to store it."""
+    """Handles a useful playthrough coming in from the frontend to store it. If
+    the playthrough already exists, it is updated in the datastore.
+    """
+
+    def __init__(self, request, response):
+        """Method that initializes member variables for the handler."""
+        super(StorePlaythroughHandler, self).__init__(request, response)
+        self.exp_issues = None
+        self.issue_schema_version = None
+
+    def _is_issue_in_exp_issues_model(
```

Comment:
I have a few comments for this method:
- The name is misleading. It does more than check if it's an issue in the list, it also returns the index when it does. You should rename it to reflect that.
- Why separate issue_type and customization_args if they're already a part of a single object (`Playthrough`?).
  - This would more sense if you pass the entire object you plan to match against, rather than pieces of it.

Reply:
Both comments make sense. Done.